storage/replica_command.go

```diff
@@ -1648,7 +1648,7 @@ func (r *Replica) applyNewLeaseLocked(
 			// in on the Lease struct.
```

Comment:
The name `StartStasis` is a bit confusing in that case. Perhaps it should be `Stasis`.